source/use-cases/home-resolver.rst

```diff
@@ -1,6 +1,200 @@
+
+.. code-block:: bash
+
+	pkill -f unbound
+
+And you can restart Unbound with:
+
+.. code-block:: bash
+
+	unbound -c /etc/unbound.conf
+
+From this point on, we can stop, start, and reload the instance with :command:`unbound-control` if you want to make changes to the configuration.
+
+Testing the resolver from a remote machine
+------------------------------------------
+
+So now we have a DNS resolver which should be reachable from within the network. To verify this we need to find the IP address of the resolver machine which can be found on the machine itself. For this tutorial we will use the address ``10.0.0.2`` (not ``127.0.0.1`` as we saw earlier) as an example. Armed with the IP address we can send a query to our DNS resolver from another machine which is within our home network. To do this we use the same dig command, only we change the IP address where the query is asked.
+
+.. code-block:: bash
+
+	dig example.com @10.0.0.2
+
+This should give the same result, including the ``SERVER`` entry, as the query from the local test above.
+
+Where it all comes together
+---------------------------
+
+We should now have a functioning DNS resolver that is accessible to all machines in our network (make sure you do before you continue). 
+
+The next step then becomes a little tricky as there are many options and variations possible. We have a choice of which machines in our network will be using our configured DNS resolver. This can range from a single machine to all the machines that are connected. Since this tutorial cannot (and does not try to) be comprehensive for the range of choices, we will look at some of the basic examples which you can implement and expand on.
+
+Most machines when they first connect to a network get a “recommended resolver” from your router using DHCP (Dynamic Host Configuration Protocol). To change this, we need to log into the router. To find the IP address of our home router we use which is likely be under :option:`default gateway`:
+
+.. code-block:: bash
+
+	ip route
+
+When you've found the IP address of your home router, you can copy the address to a web browser, which should give you access to the router configuration portal. If you can't find the portal using this method, we suggest to consulting the manual or the manufacturer's website. When you have access, you should change the default gateway to the IP address of the machine running Unbound. In the case of our example, that would be 10.0.0.2.
```

Comment:
"you should change the default gateway to the IP address of the machine running Unbound"
I think you meant that the user should change the DHCP configuration to advertise the machine running Unbound as the network DNS server.